packages/client/src/leaderRelay.ts

```
@@ -88,15 +91,32 @@ function wipeChunk(payload: unknown): void {
  * Wipes every buffer a port message carries, on the routes that answer it with
- * neither a relay nor a refusal — a write step's upload chunk, and a settings
- * command's BYO bearer.
+ * neither a relay nor a refusal — a write step's upload chunk, a settings
+ * command's BYO bearer, and a rendezvous step's secret scalars.
  */
 function wipeDropped(message: unknown): void {
-  const envelope = message as { write?: unknown; command?: unknown } | null | undefined;
+  const envelope = message as
+    | { write?: unknown; command?: unknown; read?: { step?: unknown } | null }
+    | null
+    | undefined;
```

ESLint

[error] Replace `|·{·write?:·unknown;·command?:·unknown;·read?:·{·step?:·unknown·}·|·null·}⏎····|·null⏎···` with `{·write?:·unknown;·command?:·unknown;·read?:·{·step?:·unknown·}·|·null·}·|·null`

(prettier/prettier)